examples/grid-layout.adoc

```diff
@@ -0,0 +1,100 @@
+= Grid Layout
+:docinfo: private
+:source-highlighter: highlight.js
+:highlightjs-languages: clojure,scala
+
+[.columns]
+== 2 columns
+
+[.column]
+--
+* **Edgar Allen Poe**
+* Sheri S. Tepper
+* Bill Bryson
+--
+
+[.column]
+--
+Edgar Allan Poe (/poʊ/; born Edgar Poe; January 19, 1809 – October 7, 1849) was an American writer, editor, and literary critic.
+--
+
+[.columns.wrap]
+== Multi-columns with wrap
+
+[.column]
+--
+.Kotlin
+[source,kotlin]
+----
+fun main() {
+    println("Hello, World!")
+}
+----
+--
+
+[.column]
+--
+.Clojure
+[source,clojure]
+----
+(defn -main [& args]
+  (println "Hello, World!"))
+----
+--
+
+[.column]
+--
+.Scala
+[source,scala]
+----
+object HelloWorld {
+  def main(args: Array[String]): Unit = {
+    println("Hello, world!")
+  }
+}
+----
+--
+
+[.columns]
+== Columns with size
+
+[.column.is-one-third]
+--
+* **Kotlin**
+* Java
+* Scala
+--
+
+[.column]
+--
+Programming language for Android, mobile cross-platform
+and web development, server-side, native,
+and data science. Open source forever Github.
+--
+
+[.columns]
+== 3 columns
+
+[.column]
+* Java
+* **Kotlin**
+
+[.column]
+* Node
+* **Deno**
+
+[.column]
+* Ruby
+* **Crystal**
+
+
+[.columns.is-half.green.left]
```

Comment:
The green doesn't work right now. It's missing something.

I remember background coloring being a bit tricky to get working because they need to be applied to an entirely different part of the DOM. See `examples/background-color.adoc`. Can you make sure your example works please?

Reply:
Ooops the docinfo was ignored. I've modified the `.gitignore` file and pushed `examples/grid-layout-docinfo-revealjs.html`